test_stack.py

```diff
@@ -21,10 +32,15 @@ def test_parse():
         list(stack.parse_program('horp; push 0'))
 
 
-# def test_parse_unicode():
-#     for sigil in stack.sigil_to_op:
-#         instruction = next(stack.parse_program(sigil))
-#         assert instruction == Instr(stack.sigil_to_op[sigil])
+def test_comments():
+    code = (': this is a comment\n' +
+            ': this is another; add 1')
+    assert list(stack.parse_program(code)) == []
+    code = '    : this is a comment with leading whitespace'
+    assert list(stack.parse_program(code)) == []
+    code = (':  comment\n' +
+            'add')
+    assert list(stack.parse_program(code)) == [Instr('add')]
```

Comment:
You may want to test if labels are compatible (although if we process labels before comments then that won't be necessary)